Framework/Algorithms/src/ReflectometryReductionOneAuto.cpp

```diff
@@ -462,21 +497,59 @@ void ReflectometryReductionOneAuto::exec() {
     if (theta_in.is_initialized()) {
       refRedOne->setProperty("ThetaIn", theta_in.get());
     }
+    double scaleFactor = getProperty("ScaleFactor");
+    if (scaleFactor != 1.0) {
+      refRedOne->setProperty("ScaleFactor", scaleFactor);
+    }
+    auto momentumTransferMinimum = isSet<double>("MomentumTransferMinimum");
+    auto momentumTransferStep = isSet<double>("MomentumTransferStep");
+    auto momentumTransferMaximum = isSet<double>("MomentumTransferMaximum");
 
+    if (momentumTransferStep.is_initialized()) {
```

Comment:
For future reference `boost::optional::is_initialized` is now [deprecated](http://www.boost.org/doc/libs/1_34_0/libs/optional/doc/optional.html)